internal/cmd/egctl/translate.go

```diff
@@ -274,11 +274,13 @@ func translateGatewayAPIToGatewayAPI(resources *gatewayapi.Resources) (gatewayap
 			epInvalid = true
 			msg := fmt.Sprintf("%s: %v", status.MsgGatewayClassInvalidParams, err)
 			status.SetGatewayClassAccepted(resources.GatewayClass, false, string(gwapiv1.GatewayClassReasonInvalidParameters), msg)
+			status.SetGatewayClassSupportedFeatures(resources.GatewayClass)
```

Comment:
what I meant in my previous comment was if you called `SetGatewayClassSupportedFeatures` within `status.SetGatewayClassAccepted`, its easier to manage the code in case the code updates in the future

Reply:
Thanks for the clarification. I agree entirely